spec/controllers/citizens_controller_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+RSpec.describe CitizensController do
+  describe 'GET #index' do
+    let(:citizens) { create_list(:citizen, 5) }
+
+    before do
+      get :index
+    end
+
+    it 'validate status' do
+      expect(response).to  have_http_status(:success)
+    end
+
+    it 'validate render template' do
+      expect(response).to  render_template(:index)
+    end
+
+    it 'validate citizens list' do
+      expect(assigns(:citizens)).to match_array(citizens)
```

Comment:
Eu vi que você adicionar `order` na lista de municipes que é retornada na index. Teria que testar que `citizens` está vindo ordenado por `id` conforme você implemento no controller. Com o `match_array` você não consegue garantir isso, teria que usar o `eq`, depois dá uma olhada nesse link:

https://tosbourn.com/testing-arrays-in-rspec/#when-you-care-about-order